sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1527,22 +1646,45 @@ export class CertificateClient {
     return this.getCertificateFromCertificateBundle(result._response.parsedBody);
   }
 
-  private getCertificateFromCertificateBundle(certificateBundle: CertificateBundle): Certificate {
+  private getCertificateFromCertificateBundle(
+    certificateBundle: CertificateBundle
+  ): KeyVaultCertificate {
     const parsedId = parseKeyvaultEntityIdentifier("certificates", certificateBundle.id);
 
+    const attributes: any = certificateBundle.attributes || {};
     const policy = toPublicPolicy(certificateBundle.policy || {});
 
-    const properties = {
+    let abstractProperties: any = {
+      keyId: certificateBundle.kid,
+      secretId: certificateBundle.sid,
+      name: parsedId.name,
+      createdOn: attributes.created,
+      updatedOn: attributes.updated,
+      expiresOn: attributes.expires,
       ...parsedId,
       ...certificateBundle,
```

Comment:
This `...certificateBundle` results in `kid` and `sid` being extracted out which we dont want anymore because we need them to be named `keyId` & `secretId`.

Reply:
I'm also noticing that KeyVaultCertificate and CertificateProperties have overlap in the fields they provide.  This might be something we keep for now and then clean up before the main certificates release, but it is a bit surprising.

Reply:
This has been handled in @jonathandturner's commit https://github.com/Azure/azure-sdk-for-js/pull/5991/commits/1ef22d22baf363462e791bd9064546c1208a0bdf